### DISCUSSION TITLE
How to use multiple condition shortcuts to share the same styles?

### DISCUSSION THREAD

**Discussion by fseglard** [chakra-ui/panda#1191] [Q&A] [answered]
Hi everyone!

I was wondering if there is a way to share the same styles for multiple condition shortcuts?

For example, let's assume that I want to share the same styles for `_focus` and `_expanded` condition shortcuts, I could have done something like:

```ts
const styles = css({
  '_focus, _expanded': {
    ... // My shared styles
  }
});
```

But it doesn't work, obviously, so I was wondering how can I achieve this?

**Comment by astahmer** [comment, marked as answer]
you can either make a [custom condition](https://panda-css.com/docs/customization/conditions) that combines both of those, or just re-use the same style object like:

```
const conditionalStyles = css.raw({ /* ... */ })
const styles = css({
  '_focus': conditionalStyles,
  '_expanded': conditionalStyles,
});
```

**Reply by fseglard** [reply]
Thanks @astahmer for the quick reply! I'm gonna try the custom condition!
